composer.json

```diff
@@ -17,11 +17,15 @@
         "johnkary/phpunit-speedtrap": "^1.0",
         "php-mock/php-mock-phpunit": "^1.1",
         "phpunit/phpunit": "4.8.*",
-        "squizlabs/php_codesniffer": "2.*"
+        "squizlabs/php_codesniffer": "2.*",
```

Comment:
`"squizlabs/php_codesniffer": "2.*"`

phpcs 3.0 was released yesterday but it was in pre-release for a while.  We should test with 

`"squizlabs/php_codesniffer": "*"` to get the latest release. They do have a good releasing system and stability so I don't think it would break.